src/explore-education-statistics-admin/src/pages/release/summary/ReleaseSummaryForm.tsx

```diff
@@ -87,11 +87,38 @@ const ReleaseSummaryForm = <FormValues extends EditFormValues>({
     return optGroups;
   };
 
+  const months = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+  const todaysDate = new Date();
```

Comment:
This should be constructed in the validator itself. Constructing it here could cause bugs if the user were to leave this screen open for longer than a day.